mc/gui/breathing_notification.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import time
+from PyQt5 import QtCore
+from PyQt5 import QtWidgets
+from PyQt5 import QtGui
+import mc.mc_global
+import mc.model
+
+
+class BreathingNotification(QtWidgets.QFrame):
+    # close_signal = QtCore.pyqtSignal(list, list)
+    breathe_signal = QtCore.pyqtSignal()
+    close_signal = QtCore.pyqtSignal()
+    # wait_signal = QtCore.pyqtSignal()
+
+    def __init__(self):
+        super().__init__()
+
+        self.setWindowFlags(
+            QtCore.Qt.Dialog
+            | QtCore.Qt.WindowStaysOnTopHint
+            | QtCore.Qt.FramelessWindowHint
+            | QtCore.Qt.WindowDoesNotAcceptFocus
+        )
+        # -To avoid the window getting focus we need to set both QtCore.Qt.Dialog
+        #  and QtCore.Qt.WindowDoesNotAcceptFocus (setting QtCore.Qt.Popup +
+        #  QtCore.Qt.WindowDoesNotAcceptFocus doesn't work)
+
+        # | QtCore.Qt.WindowStaysOnTopHint
+        # | QtCore.Qt.X11BypassWindowManagerHint
+
+        self.setFocusPolicy(QtCore.Qt.NoFocus)
```

Comment:
Still takes focus away from active window on Manjaro XFCE

Reply:
Hi, thank you for finding and reporting this, i will look into it (must have been there before as well since the rest notifications are the same)

Reply:
Btw can you write an issue on this problem?